packages/variable-editor/src/protocol/types.ts

```diff
@@ -28,6 +41,7 @@ export interface Client {
   data(context: DataContext): Promise<Data>;
   saveData(saveArgs: SaveArgs): Promise<ValidationMessages>;
   validate(validate: DataContext): Promise<ValidationMessages>;
+  meta<TMeta extends keyof MetaRequestTypes>(path: TMeta, args: MetaRequestTypes[TMeta][0]): Promise<MetaRequestTypes[TMeta][1]>;
```

Comment:
I also think that this one here is not needed.

Reply:
Do not understand why it is not needed. Can you explain?

Reply:
ah sorry, I think I didn't read this correctly :)